internal/app/connectconformance/test_case_library.go

```diff
@@ -430,9 +446,44 @@ func populateExpectedUnaryResponse(testCase *conformancev1.TestCase) error {
 		TimeoutMs:      convertToInt64Ptr(testCase.Request.TimeoutMs),
 	}
 
-	// If no responses are specified for unary, the service will still return a response with the
-	// request information inside (but none of the response information since it wasn't provided).
-	def := definer.GetResponseDefinition()
+	// If this is a GET test, then the request should be marshalled and in the query params
+	if testCase.Request.UseGetHttpMethod {
+		isJSON := testCase.Request.Codec == conformancev1.Codec_CODEC_JSON
+		// Build a codec based on what is used in the request
+		codec := internal.NewCodec(isJSON)
+		reqAsBytes, err := codec.MarshalStable(msg)
+		if err != nil {
+			return err
+		}
+		var value string
+		var encoding string
+		if isJSON {
+			value = string(reqAsBytes)
+			encoding = "json"
+		} else {
+			value = base64.RawURLEncoding.EncodeToString(reqAsBytes)
+			encoding = "proto"
+		}
+		reqInfo.ConnectGetInfo = &conformancev1.ConformancePayload_ConnectGetInfo{
+			QueryParams: []*conformancev1.Header{
+				{
+					Name:  "message",
+					Value: []string{value},
+				},
+				{
+					Name:  "encoding",
+					Value: []string{encoding},
+				},
+				// TODO: How should we verify the connect version parameter?
+				// Match the name "connect" and a value that is v + some number?
```

Comment:
I must always be "v1". We don't need to worry about making this more lenient until we actually support some newer version of the protocol.